### PR TITLE
[fuchsia] Rename FuchsiaExternalViewEmbedder to GfxExternalViewEmbedder

### PR DESCRIPTION
**Pull request by uysalere**
This PR renames FuchsiaExternalViewEmbedder to GfxExternalViewEmbedder because it only holds Gfx specific code after Flatland branching.

Bug: https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=64201

### CODE REVIEW

**Review by arbreng — APPROVED** [review]

File: ci/licenses_golden/licenses_flutter [hunk 1]
```diff
@@ -1425,8 +1425,8 @@ FILE: ../../../flutter/shell/platform/fuchsia/flutter/flutter_runner_product_con
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate.cc
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate.h
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate_unittests.cc
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.cc
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.h
+FILE: ../../../flutter/shell/platform/fuchsia/flutter/gfx_external_view_embedder.cc
```
Comment (arbreng) [inline ci/licenses_golden/licenses_flutter:1428]: This needs to be sorted alphabetically, thats why the license check is failing

**Review by jaeheon — APPROVED** [review]
+2 LGTM yay to less confusing code!

File: ci/licenses_golden/licenses_flutter (hunk 1 above)
Comment (uysalere) [reply ci/licenses_golden/licenses_flutter]: Done.

**Comment by uysalere** [comment]
This PR should be good to merge now.

**Comment by jaeheon** [comment]
Looks like CQ got stuck. Worsham's ProTip is to upload a "do-nothing" patch to get CQ to run again.

**Comment by uysalere** [comment]
Thanks for the tip! I pushed a rebased patch that seemed to be enough to trigger a new CQ run.

**Pull request merged**